ydb/docs/en/core/development/suggest-change.md

```diff
@@ -98,43 +98,40 @@ And copy-paste the shown token to complete the GitHub CLI configuration.
 
 YDB official repository is [https://github.com/ydb-platform/ydb](https://github.com/ydb-platform/ydb), located under the YDB organization account `ydb-platform`.
 
-Create working dir:
+To work on the YDB code changes, you need to create a fork repository under your GitHub account. Create a fork pressing `Fork` button on the official repository page.
+
+After your fork is set up, create a local git repository with two remotes:
+- official: official YDB repository, for main and stable branches
```

Comment:
why not something like upstream/origin? these names are more common

Reply:
This way we have non-default remote names for non-default scenario (default scenario is just one remote -- origin). Thus developers won't be able to (unconsciously) push somewhere they did not intend to